app/1.0/blog/2016-08-18-polymer-starter-kit-or-polymer-cli.md

```diff
@@ -0,0 +1,106 @@
+build (and share) their own CLI templates that contain all the features they want.
+
+## OK, so how do I add my own build steps?
+
+The Starter Kit relies on Polymer CLI to handle its build process. This means 
+you can just run `polymer build` and it will generate a build for you, including 
+adding a service worker. If you need to run additional build steps we recommend 
+using the 
+[Custom Build template](https://github.com/PolymerElements/generator-polymer-init-custom-build) 
+instead. This new template project provides an “escape hatch” from the Polymer 
+CLI build process by leveraging the 
+[`polymer-build`](https://github.com/Polymer/polymer-build) Node module directly 
+in its gulpfile. `polymer-build` is the underlying library that powers Polymer 
+CLI. By using it directly in Gulp, the Custom Build template is able to add 
+additional steps to its build process. These steps can include anything from 
+minifying images, to compiling SASS/LESS, or transpiling ES2015 JavaScript.
+
+To get you started the Custom Build template already contains an example for 
+minifying images, and the polymer-build docs show additional examples for 
+[uglifying JS, compressing CSS, and minifying HTML](https://github.com/Polymer/polymer-build#extracting-inlined-cssjs).
+
+Some have asked “why doesn’t the CLI just transpile my JS for me?” and indeed it 
+may someday include that feature. For now we’re starting with the bare minimum 
+number of features in the CLI and rolling in new ones as they make sense. So if 
+something is missing you can always switch to the 
+[Custom Build template](https://github.com/PolymerElements/generator-polymer-init-custom-build) 
+to fill in the gaps.
+
+## How do I migrate my current Polymer Starter Kit project?
+
+If you’re already using Polymer Starter Kit v1, and are happy with your current 
+app setup, then you likely don’t need to migrate. Starter Kit is really meant to 
+be a scaffold to get you up and running and is not intended to be a framework 
+that you have to stay in sync with.
+
+Having said that, if you would prefer to use the app-layout elements and PRPL 
+pattern features of Polymer Starter Kit v2, then you should be able to port your 
+HTML over to the new version. The page layout of Polymer Starter Kit v1 is very 
+similar to v2, with the exception that paper-* elements have been replaced by 
+app-layout elements. Once you’ve copied everything over, you can take advantage 
+of the `polymer build` command to optimize your project.
+
+If you’d also like to maintain control of your gulp build process, then you’ll 
+want to port your code over to the Custom Build template. This is the same as 
+above, with the additional step of copying over gulp tasks that you care about. 
+As mentioned previous, you can refer to the polymer-build README for 
+[examples of how to add your own gulp tasks](https://github.com/Polymer/polymer-build/pull/19/files#diff-04c6e90faac2675aa89e2176d2eec7d8R117).
```

Comment:
This is a super-weird link (to the middle of a diff of an existing README. Can we just link to the readme?